### PR TITLE
htop: add Indonesian translation

### PR DESCRIPTION
**Pull request by frostg-012**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.id/common/htop.md
Comment (kbdharun) [inline pages.id/common/htop.md:12]:
```suggestion
`htop --user {{nama_pengguna}}`
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.
I have a small suggestion for this page. ([Ref](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md))

**Comment by frostg-012** [comment]
> LGTM, Thanks for your contribution. I have a small suggestion for this page. ([Ref](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md))



> LGTM, Thanks for your contribution. I have a small suggestion for this page. ([Ref](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md))

Updated, thanks for suggestion

**Review by kbdharun — APPROVED** [review]

**Comment by kbdharun** [comment]
@frostg-012 Thanks for the request. I had already reviewed the changes. After approval from another maintainer, this PR will be merged.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**
